src/main.rs

```diff
@@ -290,6 +293,46 @@ fn get_os() -> OsType {
     OsType::Other
 }
 
+fn get_languages(
+    env_lang: Result<String, std::env::VarError>,
+    env_language: Result<String, std::env::VarError>,
```

Comment:
Note to self: Use `Option` instead

Reply:
Good point, an `Option<String>` would be better!